src/targets.rs

```diff
@@ -122,44 +122,11 @@ impl ArmArchitecture {
         use ArmArchitecture::*;
 
         match self {
-            Arm
-            | Armeb
-            | Armv4
-            | Armv4t
-            | Armv5t
-            | Armv5te
-            | Armv5tej
-            | Armv6
-            | Armv6j
-            | Armv6k
-            | Armv6z
-            | Armv6kz
-            | Armv6t2
-            | Armv6m
-            | Armv7
-            | Armv7a
-            | Armv7ve
-            | Armv7m
-            | Armv7r
-            | Armv7s
-            | Armv8
-            | Armv8a
-            | Armv8_1a
-            | Armv8_2a
-            | Armv8_3a
-            | Armv8_4a
-            | Armv8_5a
-            | Armv8mBase
-            | Armv8mMain
-            | Armv8r
-            | Armebv7r => false,
-            Thumbeb
-            | Thumbv6m
-            | Thumbv7a
-            | Thumbv7em
-            | Thumbv7m
-            | Thumbv7neon
-            | Thumbv8mBase
+            Arm | Armeb | Armv4 | Armv4t | Armv5t | Armv5te | Armv5tej | Armv6 | Armv6j
+            | Armv6k | Armv6z | Armv6kz | Armv6t2 | Armv6m | Armv7 | Armv7a | Armv7ve | Armv7m
+            | Armv7r | Armv7s | Armv8 | Armv8a | Armv8_1a | Armv8_2a | Armv8_3a | Armv8_4a
+            | Armv8_5a | Armv8mBase | Armv8mMain | Armv8r | Armebv7r => false,
+            Thumbeb | Thumbv6m | Thumbv7a | Thumbv7em | Thumbv7m | Thumbv7neon | Thumbv8mBase
```

Comment:
I find this is significantly harder to read and modify :/ Maybe it's worth using `#[rustfmt::skip]`? The main issue is that adding or changing a single variant shows a diff with 6 others as well so it's hard to find which one changed.

Reply:
Good idea. I've now added a few `#[rustfmt::skip]` directives and reverted the formatting for those.